### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by chetanguptaa**
#### What type of PR is this?
Just fixed some indentation and improved code readability!

/kind cleanup

#### What this PR does / why we need it:
It improves code readability 
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by charles-chenzz** [comment]
please rebase your branch and I think the vendor should not be modify directly.

**Comment by chetanguptaa** [comment]
> please rebase your branch and I think the vendor should not be modify directly.

hey can you help me out I'm lost here

**Comment by charles-chenzz** [comment]
> > please rebase your branch and I think the vendor should not be modify directly.
> 
> hey can you help me out I'm lost here

run git rebase on your local branch and then force push to remote.

File: pkg/kubelet/prober/prober.go [hunk 1]
```diff
@@ -112,8 +115,9 @@ func (pb *prober) probe(ctx context.Context, probeType probeType, pod *v1.Pod, s
 	if result == probe.Warning {
 		pb.recordContainerEvent(pod, &container, v1.EventTypeWarning, events.ContainerProbeWarning, "%s probe warning: %s", probeType, output)
 		klog.V(3).InfoS("Probe succeeded with a warning", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name, "output", output)
-	} else {
-		klog.V(3).InfoS("Probe succeeded", "probeType", probeType, "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
```
Comment (bart0sh) [inline pkg/kubelet/prober/prober.go:116]: This log entry is lost. Please, return it back.

Comment (chetanguptaa) [reply pkg/kubelet/prober/prober.go:116]: Ohh I think it's done now !!

**Comment by bart0sh** [comment]
@chetanguptaa you shouldn't change anything in vendor/ directory. And please squash your commits in one, thanks.

**Pull request closed without merging**